editor/reducer.js

```diff
@@ -16,6 +16,14 @@ import { getBlockTypes } from '@wordpress/blocks';
 import { combineUndoableReducers } from './utils/undoable-reducer';
 
 /**
+* Collaboration Dependencies
+*/
+import GRTC from '../grtc/app';
+
+const isMobile = window.innerWidth < 782;
```

Comment:
This doesn't seem a good place for this.

Reply:
Both constant were never used. I removed them.